src/controllers/services.ts

```diff
@@ -62,13 +68,24 @@ export const ServicePayload = t.partial({
   service_metadata: ServiceMetadata,
   service_name: ServiceName
 });
+
 export type ServicePayload = t.TypeOf<typeof ServicePayload>;
 
 export const notificationApiClient = APIClient(
   config.adminApiUrl,
   config.adminApiKey
 );
 
+const ReviewStatus = t.partial({
+  comment: t.any,
+  detail: t.string,
+  labels: t.any,
+  status: t.number,
+  title: t.string
+});
+
+type ReviewStatus = t.TypeOf<typeof ReviewStatus>;
```

Comment:
This type seems too generic. `comment` and `labels` have a `any` type that means no validation.
```suggestion
const ReviewStatus = t.partial({
  comment: // The comment type,
  detail: NonEmptyString,
  labels: t.readonlyArray(NonEmptyString),
  status: t.number, // Only accepted status values
  title: NonEmptyString
});
type ReviewStatus = t.TypeOf<typeof ReviewStatus>;
```

Reply:
Fix with a95d264